core/iwasm/interpreter/wasm_loader.c

```diff
@@ -51,6 +51,20 @@ has_module_memory64(WASMModule *module)
 
     return false;
 }
+
+static bool
+is_table_64bit(WASMModule *module, uint32 table_idx)
+{
+    if (module->import_table_count > 0
+        && table_idx < module->import_table_count)
+        return (module->import_tables[table_idx].u.table.table_type.flags
+                & TABLE64_FLAG)
+               != 0;
+    else if (module->table_count > 0)
+        return (module->tables[table_idx].table_type.flags & TABLE64_FLAG) != 0;
```

Comment:
Seems the table_idx has been checked before? Or we should set error buf before returning false. How about just:
```
    if (table_idx < module->import_table_count)
        return (module->import_tables[table_idx].u.table.table_type.flags
                & TABLE64_FLAG)
               != 0;
    else
        return (module->tables[table_idx].table_type.flags & TABLE64_FLAG) != 0;
```